test/runnable/test_cdvecfill.d

```diff
@@ -123,730 +123,830 @@ alias AliasSeq(Args...) = Args;
 // dfmt off
 alias baselineCases = AliasSeq!(
     Code!(ubyte, 16 / ubyte.sizeof)([
-        /* push   rax                     */ 0x50,
+        /* push   rbp                     */ 0x55,
+        /* mov    rbp,rsp                 */ 0x48, 0x8b, 0xec,
         /* movd   xmm0,edi                */ 0x66, 0x0f, 0x6e, 0xc7,
         /* punpcklbw xmm0,xmm0            */ 0x66, 0x0f, 0x60, 0xc0,
         /* punpcklwd xmm0,xmm0            */ 0x66, 0x0f, 0x61, 0xc0,
         /* pshufd xmm0,xmm0,0x0           */ 0x66, 0x0f, 0x70, 0xc0, 0x00,
-        /* pop    rcx                     */ 0x59,
+        /* pop    rbp                     */ 0x5d,
         /* ret                            */ 0xc3,
     ]),
```

Comment:
If `PIC=0` won't that cause this test to fail if we accept these changes?

Reply:
I'm not sure. I just cherry-picked @wilzbach's fix to try to get the test suite to pass on my machine.
It seems this will need more work.

Reply:
I think you may need to keep both tests:  This one for -fPIC and the test without these changes for Non-PIC.  That is, if you intend to allow both PIC and non-PIC compiles, which appears to be  the case, unless I'm misunderstanding.

Reply:
How about copy/pasting this file and using `version(D_PIC) {} else` in the other? This has the advantage that the update script would still work.